programmers/카펫.js

```diff
@@ -0,0 +1,16 @@
+function solution(brown, yellow) {
+    const result = [];
+    let length  = 0;
+    
+    for (let i = 1; i < yellow + 1; i++) {
```

Comment:
```suggestion
    for (let i = 1; i < yellow + 1; i += 1) {
```
단항 ++및 --연산자는 자동 세미콜론 삽입의 대상이 되기 때문에 ,
공백으로 인해서 소스 코드의 의미를 변경이 될 수 있다고 합니다.